src/analyze/AsYouType.re

```diff
@@ -171,7 +171,7 @@ let getAst = (~cacheLocation, ~compilerVersion, ~moduleName, ~uri) => {
   Ok("NVM")
 }; */
 
-let process = (~uri, ~moduleName, ~basePath, ~reasonFormat, text, ~cacheLocation, ~compilerVersion, ~allLocations, compilerPath, refmtPath, includes, flags) => {
+let process = (~uri, ~moduleName, ~basePath, ~reasonFormat, text, ~cacheLocation, ~compilerVersion, ~allLocations, compilerPath, refmtPath, includes, flags, converter) => {
```

Comment:
Providing a converter as argument is supposed to make sure that documentation comments for `local` symbols also display text in the correct format. Previously the doc comment text was passed around as raw, so that displaying only worked for `markdown` comments, not for `odoc`